config/locales/en.yml

```diff
@@ -1208,6 +1207,7 @@ en:
     about:
       default_about_text_title: "What is Sharetribe?"
       default_about_text: "This marketplace is powered by Sharetribe platform. With Sharetribe you can easily create your own marketplace website. It's free and only takes a minute. %{click_here_link} to learn more!"
+      default_how_to_use_text: "<h1>How it works</h1><div>Here you can find information about how this site works.</div>"
```

Comment:
How about this?

```
# en.yml
default_how_to_use_title: "How it works"
default_how_to_use_content: "Here you can find information about how this site works."
```

```
...
- else
  - content = "<h1>#{t('infos.about.default_how_to_use_title')}</h1><div>#{t('infos.about.default_how_to_use_content')}</div>"
...
```

It's not allowed to add any HTML/HAML markup to translation files. Ever. No exceptions. It has already caused us quite a lot of trouble (and will cause more when we have more languages and translators) and we don't want to repeat those mistakes 😉